README.md

```diff
@@ -36,7 +36,8 @@ Please read [CONTRIBUTING](/CONTRIBUTING.md).
   - [PS4](#ps4)
   - [PSP](#psp)
   - [PSVita](#psvita)
-
+- [ZXSpectrum](#zxspectrum)
```

Comment:
I think "ZX Spectrum" looks better.

Reply:
I have changed to ZX-Spectrum because it is more standard one for the platform, I just thought that all item must not have `-` chars